manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml

```diff
@@ -13,6 +13,7 @@ images:
   viewercrd: gcr.io/ml-pipeline/google/pipelines/viewercrd:0.1.31
   visualizationserver: gcr.io/ml-pipeline/google/pipelines/visualizationserver:0.1.31
   metadataenvoy: gcr.io/ml-pipeline/google/pipelines/metadataenvoy:0.1.31
+  metadatawriter: gcr.io/ml-pipeline/google/pipelines/metadatawriter:0.1.40
```

Comment:
I think we should also add to the oncall duty to update/release this one.

Reply:
Just one time or regularly?

Reply:
Regularly maybe? IMO all the components of KFP should be maintained on a regular basis.

Reply:
Talked offline. The issue is resolved.